### PR TITLE
Mjml

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/templates/token.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        <MjmlFont
+          name="Space Grotesk"
+          href="https://fonts.googleapis.com/css2?family=Space+Grotesk"
+        />
+        <MjmlFont
+          name="JetBrains Mono"
+          href="https://fonts.googleapis.com/css2?family=JetBrains+Mono"
+        />
+        <MjmlAttributes>
+          <MjmlClass name="text-accent" color={colors['accent']} />
```
Comment (T-Damer) [inline src/templates/token.tsx]: Can't we access the property with `colors.accent`?

File: src/templates/token.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+              background: conic-gradient(
+                  from 90deg at 0.06rem 0.06rem,
+                  #0d0030 90deg,
+                  #1a0259 0
+                )
+                0 0/3rem 3rem;
+            }
+        `}</MjmlStyle>
+      </MjmlHead>
+      <MjmlBody cssClass="body" backgroundColor="#0d0030">
```
Comment (T-Damer) [inline src/templates/token.tsx]: Let's add `#0d0030` to `colors` as well, use `sc.xyz` config as reference

File: src/templates/token.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            name="font-accent"
+            fontFamily="Space Grotesk, sans-serif"
+          />
+          <MjmlClass
+            name="font-primary"
+            fontFamily="JetBrains Mono, sans-serif"
+          />
+          <MjmlAll padding="0" />
+        </MjmlAttributes>
+        <MjmlStyle>{`
```
Comment (T-Damer) [inline src/templates/token.tsx]: Can we extract those styles into `const`? It will be easier to read it

File: src/templates/token.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+            .body {
+              margin: 0;
+              padding: 0 20px;
+              -webkit-font-smoothing: antialiased;
+              -moz-osx-font-smoothing: grayscale;
+              background-color: #0d0030;
```
Comment (T-Damer) [inline src/templates/token.tsx]: Can we use variable here? Maybe css `--var()`?

File: src/templates/token.tsx [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+                  #0d0030 90deg,
+                  #1a0259 0
+                )
+                0 0/3rem 3rem;
+            }
+        `}</MjmlStyle>
+      </MjmlHead>
+      <MjmlBody cssClass="body" backgroundColor="#0d0030">
+        <MjmlSection>
+          <MjmlColumn paddingBottom="32px" paddingTop="32px">
```
Comment (T-Damer) [inline src/templates/token.tsx:73]: Maybe we can use `rem` instead of `px`?

File: src/templates/token.tsx [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+  MjmlAttributes,
+  MjmlClass,
+  MjmlAll,
+  MjmlFont,
+  Mjml2HtmlOptions,
+} from 'mjml-react'
+
+import { render } from 'mjml-react'
+
+const colors = {
```
Comment (T-Damer) [inline src/templates/token.tsx]: Maybe it's better to store this as a file, so we can use it later

**Review by T-Damer — CHANGES_REQUESTED** [review]
Really cool, let's polish it :)

File: src/templates/token.tsx (hunk 4 above)
Comment (MixailE) [reply src/templates/token.tsx]: This is not supported by gmail https://www.caniemail.com/search/?s=var

File: src/templates/token.tsx (hunk 5 above)
Comment (MixailE) [reply src/templates/token.tsx:73]: Let's fix it in the next release)

**Review by T-Damer — APPROVED** [review]
lgtm

**Pull request merged**